vagrant_config.json

```diff
@@ -139,9 +139,20 @@
           "action": "add" // one of add, remove. Default: add.
```

Comment:
quite inconsistent for data_hash['vagrant']['config_paths'] that dev-setup entry is ignoreable while other entry is fatal error.